### DISCUSSION TITLE
v1.2.3-beta14

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#506] [Announcements/公告]
**Full Changelog**: https://github.com/constanline/XQuickEnergy/compare/v1.2.3-beta12...v1.2.3-beta14

<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.2.3-beta14'>v1.2.3-beta14</a>.</em>

**Comment by 244276041** [comment]
求更新 网商银行----福利（领取福利金和赚福利金）可以抵扣信用卡和消费抵扣使用

**Reply by 706984971** [reply]
什么时候更新

**Comment by suxia666888** [comment]
老哥们这个版本更新了什么呀

**Reply by hellbearchg** [reply]
增加欢乐揍小鸡，增加小鸡使用特殊食品开关(默认关)，修复切换账号/下次循环的时候不会自动加载配置的问题

**Comment by yanxinzi** [comment]
这个版本不会做阅读书籍任务和赚消费金任务，换回 v1.2.3-beta12 就没问题

**Reply by 947814157** [reply]
换回好像也不会了今天才出现的

**Reply by lanyuliufan** [reply]
从今天开始读书获能量出问题了，b12和b14都是每次阅读只能获取0克能量，只是b12可以从早8点就触发，b14要下午才能触发，期待大佬修复

**Comment by Tuz-001** [comment]
一直弹窗拒绝访问检查是否使用代理，是需要设置哪里呀？

**Reply by AObama555** [reply]
我跟你一样

**Reply by tianyu8867** [reply]
我也一样，不确定什么时候弹出来，只能临时关掉模块使用

**Reply by 184377** [reply]
消费金的问题

**Reply by rkdawn** [reply]
> 消费金的问题

消费金？是哪个

**Comment by w54342162** [comment]
我的芝麻粒开启不了三项功能：
1、读书听书
2、支付宝消费金
3、支小镇

**Reply by w54342162** [reply]
支付是最新版本

**Comment by xzcgg** [comment]
这个版本的阅读好像下午才能启用，是否可以不加限制阅读时间，或者可以自定义时间

**Reply by yanxinzi** [reply]
确定会做阅读任务吗，我还以为这个版本不会做，就先用回上个版本了

**Reply by acooler15** [reply]
如果关闭消费金的话，应该就没问题了，在读取最后执行时间的时候，大佬错误的使用了消费金的执行时间。执行间隔好像是6个小时。

**Reply by xzcgg** [reply]
期待大佬下个版本修复，取消阅读时间限制，或者自定义时间段✊

**Reply by 947814157** [reply]
看了你们的留言我决定不更了

**Reply by 947814157** [reply]
上个版本也不做阅读了![IMG_20240227_125332.jpg](https://github.com/constanline/XQuickEnergy/assets/102625983/72ccebd3-540d-4435-871a-20b4df4351fb)我看了结果是这样的更新完之后目前还是不行不知道你说的下午是什么时候

**Reply by lanyuliufan** [reply]
从今天开始读书获能量出问题了，b12和b14都是每次阅读只能获取0克能量，只是b12可以从早8点就触发，b14要下午才能触发，期待大佬修复

**Comment by wrym003** [comment]
大佬，这个版本的神奇海洋，没有办法完成帮好友清理垃圾任务，总是垃圾清理完毕但是任务没有完成

**Comment by mealx** [comment]
大佬，这个版本我清除通知的时候芝麻粒也会一起被清除，然后点进支付宝，芝麻粒的通知又会出现，请问是什么问题啊。我同时开了水晶1.55，水晶的通知就一直是固定的。

**Reply by 947814157** [reply]
水晶不是停更了嘛还能用啊而且水晶貌似没有芝麻粒这么全哦

**Reply by mealx** [reply]
可以用，用水晶来保活，我把芝麻粒版本退回到beta12，再配合水晶已经完美使用了

**Comment by sundongxiao** [comment]
水晶可以雇佣小鸡呐
&nbsp;


Dongxiao.Sun
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月29日(星期四) 下午5:08
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: Re: [constanline/XQuickEnergy] v1.2.3-beta14 (Discussion #506)





 
水晶不是停更了嘛还能用啊而且水晶貌似没有芝麻粒这么全哦

**Comment by 947814157** [comment]
水晶的地址可以给一个吗研究研究



---Original---
From: ***@***.***&gt;
Date: Thu, Feb 29, 2024 18:32 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [constanline/XQuickEnergy] v1.2.3-beta14 (Discussion #506)




 水晶可以雇佣小鸡呐 
 &amp;nbsp; 
 
 
 Dongxiao.Sun 
 ***@***.*** 
 
 
 
 &amp;nbsp; 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人: ***@***.***&amp;gt;;  
 发送时间: 2024年2月29日(星期四) 下午5:08 
 收件人: ***@***.***&amp;gt;;  
 抄送: ***@***.***&amp;gt;;  
 主题: Re: [constanline/XQuickEnergy] v1.2.3-beta14 (Discussion #506) 
 
 
 
 
 
   
 水晶不是停更了嘛还能用啊而且水晶貌似没有芝麻粒这么全哦

**Reply by Tuz-001** [reply]
https://www.123pan.com/s/ZQUiVv-DBPxh.html提取码:zAT0
我有安装包

**Comment by 947814157** [comment]
水晶什么版本有地址吗我试试看看





---Original---
From: ***@***.***&gt;
Date: Fri, Mar 1, 2024 22:52 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [constanline/XQuickEnergy] v1.2.3-beta14 (Discussion #506)




 
可以用，用水晶来保活，我把芝麻粒版本退回到beta12，再配合水晶已经完美使用了

**Comment by xzcgg** [comment]
关于神奇物种卡片赠送功能，发现了只要在芝麻粒翻卡前手动翻卡，或者别的科技先翻了卡，这个赠送功能就不启用，不知道其他人有没有这种问题

**Comment by xiaolian0926** [comment]
大佬，海洋碎片什么时候添加个白名单，谢谢

**Comment by haffers** [comment]
希望大佬有空的时候可以增加一下好友运动，没什么福利，就是单纯的做公益，产生和捐运动币的，默认举杠铃的性价比是最高的，功能就两个，一个是定时让好友运动，另一个是抢好友，大佬好人一生平安，慢慢来，加油。

**Reply by doooq** [reply]
支持😁，XQ之前那个就不错

**Comment by 947814157** [comment]
非常感谢要是有项目地址或者TG官方频道就更好了



---Original---
From: ***@***.***&gt;
Date: Mon, Mar 25, 2024 09:05 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [constanline/XQuickEnergy] v1.2.3-beta14 (Discussion #506)




 
https://www.123pan.com/s/ZQUiVv-DBPxh.html提取码:zAT0
 我有安装包

**Comment by Evelutch** [comment]
是不是凉了啊     近期操作异常了😂😂

**Reply by VictoireWood** [reply]
今天我也是突然显示异常。

**Reply by rainbowshine521** [reply]
我的异常一个星期了。。。

**Reply by lanyuliufan** [reply]
我也是从4月3日早上开始提示“您近期操作存在异常，请稍后再试”。支付宝客服回复是发现账号使用外挂，目前只是不能收能量，其它功能正常，期间有过几次行走能量被外挂收取。

**Comment by 947814157** [comment]
不止一个人像咱这样的呢，大佬也不更新了，偶尔会收那么点点，期待新版本吧，阅读的任务都直接没有了估计也是这个的缘故



---Original---
From: ***@***.***&gt;
Date: Fri, Apr 12, 2024 16:21 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [constanline/XQuickEnergy] v1.2.3-beta14 (Discussion #506)




 
我也是从4月3日早上开始提示“您近期操作存在异常，请稍后再试”。支付宝客服回复是发现账号使用外挂，目前只是不能收能量，其它功能正常，期间有过几次行走能量被外挂收取。

**Reply by haffers** [reply]
把支付宝退出再打开就能收能量了，我的阅读任务也没了，弹窗代理提示不影响收能量、喂鸡、摆摊、农场那些的正常运作

**Comment by 706984971** [comment]
开始提示“您近期操作存在异常，请稍后再试”。支付宝客服回复是发现账号使用外挂，目前只是不能收能量，作者大哥大哥

**Reply by haffers** [reply]
我这边只是弹窗拒绝访问检查是否使用代理，不过并不影响收能量，你这个提示的话看着有可能被检测到root或者被检测到有用插件，要不先看看隐藏root有没有漏了

**Comment by haffers** [comment]
操作异常的有可能是被检测到了使用插件或者root环境，可以试试检测一下手机的隐藏root、Zygisk（Zygisk Next）、shamiko白名单和HMA有没有做好，跑个Applist Detector、Ruru、和Momo看看有没有问题，作者更新之前我们先等等吧，目前我这边也只是提示“弹窗拒绝访问检查是否使用代理”，森林、海洋、农场、庄园、新村、小镇、经营、运动那些都还在正常运行着，如果突然不行了，可以试试支付宝关了再开

**Reply by 706984971** [reply]
我root没有任何问题，我数字人民币 各大银行app打开都没有问题，以及其他检测都没有问题

**Reply by haffers** [reply]
厉害了，严谨，那有可能是支付宝检测的先后顺序的问题了，那个提示“弹窗拒绝访问检查是否使用代理”的时候，也是我小号先弹出，我的大号过了好几天才有的，这个操作异常看来了我也避不开了哈哈哈，那就期待作者大佬发力更新了

**Comment by 706984971** [comment]
更新啊

**Comment by w54342162** [comment]
增加个绿色里程签到呗。这个现在只能手动签到

**Comment by Ai686Leo** [comment]
感谢！因为你是无偿维护

**Comment by doooq** [comment]
装的最新支付宝和最新芝麻粒14，但进zfb然后会卡顿，像一直在重复一个动作的点击，我点个程序，要迅速点击两下才能成功，一个卡顿

**Reply by xzcgg** [reply]
如果是内置的，那可能会有这种问题，等使用一段时间后就应该会好点，或者换一下ZFB版本

**Reply by luqiuyin** [reply]
去哪里下载内置的？所谓内置是免root吗？

**Reply by doooq** [reply]
> 去哪里下载内置的？所谓内置是免root吗？

内置是自己去嵌入，是无root

**Comment by doooq** [comment]
获得饲料 0g，这个是bug吗![Screenshot_2024-04-29-08-27-34-80_21bfa4bafe1dae3d40f43a3eaf21a68f.jpg](https://github.com/constanline/XQuickEnergy/assets/168314987/6cfff5a7-bb18-49d8-8fe3-de91817c06fc)

**Reply by haffers** [reply]
暂时还没碰到，请问你是免ROOT环境吗？

**Reply by doooq** [reply]
无root，自己嵌入的

**Reply by luqiuyin** [reply]
怎么自己嵌入啊？有相关教程吗？跪谢

**Reply by luqiuyin** [reply]
怎么自己嵌入？去哪学啊？

**Reply by haffers** [reply]
越来越多软件会做ROOT环境检测和模块检测了，不排除你是被检测到模块使用了。我懂得不多，也许你可以尝试搜一下免ROOT怎么做隐藏

**Reply by doooq** [reply]
> 怎么自己嵌入？去哪学啊？

兄弟啊，我有视频教程，怎么给你咧

**Reply by doooq** [reply]
> 越来越多软件会做ROOT环境检测和模块检测了，不排除你是被检测到模块使用了。我懂得不多，也许你可以尝试搜一下免ROOT怎么做隐藏

现在看了一下，这个辅助的记录是获得0饲料，但实际还是领到了的，不影响

**Reply by luqiuyin** [reply]
可以发链接吗？或者给点线索我自己去搜索，我现在是完全不知道这是啥。。

**Reply by haffers** [reply]
我了解的不多，不过我可以拿芝麻粒举例子。前期准备：github搜索shizuku和lspatch下载安装，安装支付宝和芝麻粒，不过lspatch已经好一段时间不更新了，你可以试试Npatch。

操作：
→打开开发者选项→连接WiFi→打开shizuku→开发者选项→进入无线调试→在状态栏里的shizuku输入无线调试的码→使用配对码配对→成功链接后后退到shizuku→启动→授权给lspatch→打开lspatch→管理→加号→选择已安装的应用程序→支付宝→集成模式→嵌入模块→已安装应用程序→芝麻粒→确定→开始修补→确认卸载→确认安装

**Reply by haffers** [reply]
magisk、lsposed都有github、QQ频道和其他讨论区，找找你会发现新大陆

**Reply by haffers** [reply]
确实，领的到就行

**Reply by doooq** [reply]
> 可以发链接吗？或者给点线索我自己去搜索，我现在是完全不知道这是啥。。

通过网盘分享的文件：lsp.apk等2个文件
链接:https://pan.baidu.com/s/1uN3Q9BuRnXlQyHhsfvuJJA?pwd=unqb 
提取码:unqb
复制这段内容打开「网盘APP 即可获取」，兄弟，这个文件里面

**Comment by haffers** [comment]
最近蚂蚁庄园加了个做任务得三套服装的活动，一开始我还以为是一次，结果现在成了常态。希望作者能把这个列入更新计划，谢谢，目前多数功能还是处于正常运行的状态的。
已ROOT，magisk+zygisk+LSPosed+shamiko白名单+HMA
支付宝版本：10.5.78.8000

**Comment by haffers** [comment]
致大家：芝麻信用-生活记录里有好几个可以持续做(自动完成任务且自动重新领取新任务)的任务,目前芝麻粒已经能让它自主完成蚂蚁庄园、蚂蚁森林、蚂蚁新村、芭芭农场，大家去那里直接打开任务就行，芝麻粒会帮大家自动做任务的。

致作者：目前防骗卡和保障金也支持“持续做”功能了，作者如果更新的话希望能把这两个的任务也考虑一下，谢谢
